README.md

```diff
@@ -10,6 +10,16 @@ The Devfile Registry operator manages the lifecycle of the following custom reso
 
 Issue tracking repo: https://github.com/devfile/api with label area/registry
 
+## Deployment Requirements
+
+Deployment cluster must meet one of the following criteria:
+
+- OpenShift Container Platform (OCP) 4.12.x
+- Kubernetes 1.19.x-1.24.x
```

Comment:
I don't know if the operator can support these versions.  We bumped up the k8s api version to v0.26 but I don't know what the backward compatibility statement is there.  We should check before making this statement.

Reply:
@kim-tsao Looks like k8s api version v0.26.x targets Kubernetes v1.26.x if I am understanding the versions correctly here: https://github.com/kubernetes/api/releases/tag/v0.26.4

Reply:
@michael-valdron yes it looks like that is the case.  Something to keep in mind when we're upgrading these dependencies and their impact to Openshift - we need to create a matrix doc somewhere.  

- [Openshift 4.12 supports Kubernetes 1.25]( https://docs.openshift.com/container-platform/4.12/release_notes/ocp-4-12-release-notes.html#ocp-4-12-admin-ack-upgrading)
- [Kubernetes supported versions and skew policies](https://kubernetes.io/releases/version-skew-policy/#supported-versions)

The k8s api version should be backwards compatible with Openshift since it's only one minor release difference.  We should try to align with Openshift releases to ensure we don't move up too far ahead.

Reply:
> @michael-valdron yes it looks like that is the case. Something to keep in mind when we're upgrading these dependencies and their impact to Openshift - we need to create a matrix doc somewhere.
> 
>     * [Openshift 4.12 supports Kubernetes 1.25](https://docs.openshift.com/container-platform/4.12/release_notes/ocp-4-12-release-notes.html#ocp-4-12-admin-ack-upgrading)
> 
>     * [Kubernetes supported versions and skew policies](https://kubernetes.io/releases/version-skew-policy/#supported-versions)
> 
> 
> The k8s api version should be backwards compatible with Openshift since it's only one minor release difference. We should try to align with Openshift releases to ensure we don't move up too far ahead.

@kim-tsao I requested a re-review for the matrix doc I created: https://github.com/michael-valdron/devfile-registry-operator/blob/8eb11f5f80579e672bbf770249b68e83fbd38b73/CLUSTER_SUPPORT.md

Reply:
Thanks @michael-valdron! /lgtm